tests/vm

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
As a new test, it needs to be added to `.github/workflows/tests.yml` in the long list of tests we run. Please put it alphabetically sorted in the list.